packages/http/src/validator/deserializer/HttpParamDeserializerRegistry.ts

```diff
@@ -0,0 +1,19 @@
+import { IHttpParamDeserializerRegistry } from './IHttpParamDeserializerRegistry';
```

Comment:
I don't think it could be done. This class is a generic registry for style deserializers.

Please see index.ts. It contains two instances of it:
- preconfigured 'headers' deserializers registry and
- preconfigured 'query' deserializers registry